app/controllers/idv/gpo_verify_controller.rb

```diff
@@ -4,11 +4,18 @@ class GpoVerifyController < ApplicationController
     include StepIndicatorConcern
     include FraudReviewConcern
 
+    prepend_before_action :note_if_user_did_not_receive_letter
```

Comment:
Nice trick here!

Reply:
Does `before_action` work here rather than `prepend_before_action`?

Reply:
No, the `IdvSession` concern redirects before this `before_action` would execute.